Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM registry.redhat.io/rhel8/go-toolset:1.15 AS builder
+FROM registry.redhat.io/rhel8/go-toolset:1.20.12-5.1712568462 AS builder
 WORKDIR $GOPATH/src/mypackage/myapp/
 COPY . .
 ENV GO111MODULE=on
 USER root
 RUN go get -d -v
 RUN make validate
 RUN make test
-RUN CGO_ENABLED=0 go build -o /go/bin/quickstarts
+RUN CGO_ENABLED=0 go build -buildvcs=false -o /go/bin/quickstarts
 
```

Comment:
We are using the latest version of the package. Nothing we can update to. @e can't use rbal 9 yet.

Reply:
I think the path is `registry.redhat.io/ubi8/ubi:latest`, I see it erroring being able to fetch the current one

Reply:
That's because the image is not accessible outside of the internal network :( 

This is a known issue. The builds on GH actions will be failing for now.